db/index.js

```diff
@@ -1,8 +1,9 @@
 const low = require('lowdb');
 const FileSync = require('lowdb/adapters/FileSync');
 const createModel = require('./models');
+const { dirname, resolve } = require('path');
 
-const adapter = new FileSync('db/db.json');
+const adapter = new FileSync(resolve(dirname(''), 'db/db.json'));
```

Comment:
did you add this because db.json was not found on PROD?

Reply:
yup.